target/linux/apm821xx/sata/config-default

```diff
@@ -1,43 +1,52 @@
 CONFIG_APOLLO3G=y
 CONFIG_ATA=y
-CONFIG_ATA_BMDMA=y
-CONFIG_ATA_SFF=y
+CONFIG_BCM_NET_PHYLIB=y
 CONFIG_BLK_DEV_DM=y
 CONFIG_BLK_DEV_DM_BUILTIN=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_MD=y
 CONFIG_BLK_DEV_SD=y
+CONFIG_BLK_MQ_STACKING=y
 CONFIG_BLK_PM=y
+CONFIG_BLOCK_HOLDER_DEPRECATED=y
+CONFIG_BLOCK_LEGACY_AUTOLOAD=y
 CONFIG_BROADCOM_PHY=y
+CONFIG_BUFFER_HEAD=y
+CONFIG_CRYPTO_CRC32=y
+CONFIG_CRYPTO_CRC32C=y
 CONFIG_CRYPTO_MD5_PPC=y
 CONFIG_CRYPTO_SHA1_PPC=y
++# CONFIG_DM_CRYPT is not set
++# CONFIG_DM_INIT is not set
++# CONFIG_DM_MIRROR is not set
++# CONFIG_DM_SNAPSHOT is not set
```

Comment:
@neheb There is an extra `+` in front. Should we add these options? Please create a PR to fix this.

Reply:
hrm copy/paste mistake probably. Will fix.